src/experimental/patterns/ec2-app.ts

```diff
@@ -9,10 +9,29 @@ import type { GuEc2AppProps } from "../../patterns";
 import { GuEc2App } from "../../patterns";
 import { isSingletonPresentInStack } from "../../utils/singleton";
 
+interface AutoScalingRollingUpdateDurations {
+  /**
+   * The time between each check of an instance's health within the target group.
+   */
+  sleep: Duration;
+
+  /**
+   * Additional time to cover the time spent polling an instance's health within the target group before sending the CloudFormation signal.
+   */
+  buffer: Duration;
+}
+
+export const RollingUpdateDurations: AutoScalingRollingUpdateDurations = {
+  sleep: Duration.seconds(5),
+  buffer: Duration.minutes(1),
```

Comment:
I wonder if we can be more specific with this value by making it relative to the target group's health check?

For example [HealthCheckIntervalSeconds](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-healthcheckintervalseconds) * [HealthCheckTimeoutSeconds](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-healthchecktimeoutseconds) * [HealthyThresholdCount](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-healthythresholdcount).